lib/internal/abort_controller.js

```diff
@@ -95,8 +95,21 @@ const dependantSignalsCleanupRegistry = new SafeFinalizationRegistry((signalWeak
     }
   });
 });
+
 const gcPersistentSignals = new SafeSet();
 
+const finalizer = new SafeFinalizationRegistry(({ sourceSignalRef, composedSignalRef }) => {
```

Comment:
This could have a more descriptive name

Reply:
will improve it. Maybe with something like `persistentSignalsFinalizer`?

Reply:
I would suggest `sourceSignalsCleanupRegistry`: Just like `dependantSignalsCleanupRegistry` cleans up `signal[kDependantSignals]`, `sourceSignalsCleanupRegistry` cleans up `signal[kSourceSignals]`.